cdap-ui/app/features/dashboard/controllers/dashboard-ctrl.js

```diff
@@ -105,16 +105,20 @@ function ($scope, $state, $dropdown, rDashboardsModel, MY_CONFIG) {
   }
 
   $scope.changeColumn = function (n) {
-    $scope.currentBoard.changeColumn(n);
```

Comment:
Should we remove this attribute off the scope object, if it can be outdated?
Alternative is to keep it updated?